src/Agent/NewRelic/Agent/Core/Logging/LoggerBootstrapper.cs

```diff
@@ -107,27 +108,38 @@ private static LoggerConfiguration ConfigureInMemoryLogSink(this LoggerConfigura
         }
 
         /// <summary>
-        /// Add the Event Log sink if running on .NET Framework
+        /// Add the Event Log sink if running on Windows
         /// </summary>
         /// <param name="loggerConfiguration"></param>
         private static LoggerConfiguration ConfigureEventLogSink(this LoggerConfiguration loggerConfiguration)
         {
-#if NETFRAMEWORK
-            const string eventLogName = "Application";
-            const string eventLogSourceName = "New Relic .NET Agent";
+#if NETSTANDARD2_0
```

Comment:
We also want this on Framework, right?

Reply:
Oh yeah, I guess I just need to #ifdef around the OS checking. Good catch.